src/components/widgets/sales-opportunities-timeline/sales-opportunities-timeline.tmpl.html

```diff
@@ -0,0 +1,95 @@
+            <div class="col-xs-8 sortable-title" ng-click="sort('opportunity')">
+              Opportunity Timeline
+              <i ng-show="sortedColumn == 'opportunity'" class="fa" ng-class="ascending ? 'fa-sort-alpha-asc' : 'fa-sort-alpha-desc'" />
+            </div>
+            <div class="col-xs-2 text-right sortable-title" ng-click="sort('total')">
+              Total
+              <i ng-show="sortedColumn == 'total'" class="fa" ng-class="ascending ? 'fa-sort-alpha-asc' : 'fa-sort-alpha-desc'" />
+            </div>
+            <div class="col-xs-2 text-right sortable-title" ng-click="sort('avg')">
+              Avg
+              <i ng-show="sortedColumn == 'avg'" class="fa" ng-class="ascending ? 'fa-sort-alpha-asc' : 'fa-sort-alpha-desc'" />
+            </div>
+          </div>
+
+          <!-- Groups of statements with their accounts -->
+          <div class="row lines-group" ng-repeat="statement in widget.content.opportunities | filter: filterBySalesStage" >
+            <div class="col-xs-12">
+              <div class="row widget-line" ng-click="toggleSelectedElement(statement)">
+                <div class="col-xs-8">{{statement.name}}</div>
+                <div class="col-xs-2 text-right">{{statement.total_duration + 'd'}}</div>
+                <div class="col-xs-2 text-right">{{statement.avg_duration + 'd'}}</div>
+                <div class="selection-tag" ng-if="isSelected(statement)" style="background-color: {{ getSelectLineColor() }}"/>
+              </div>
+            </div>
+
+          </div>
+        </div>
+      </div>
+
+      <div class="col-md-6 right-panel text-center" ng-show="widget.isExpanded()">
+
+        <div ng-show="selectedElement">
+
+          <h4>{{selectedElement.name | titleize}}</h4>
+
+          <div class="chart-container">
+            <div impac-chart draw-trigger="::drawTrigger.promise" deferred="::chartDeferred"></div>
+          </div>
+
+          <div class="widget-lines-container">
+            <div class="row widget-line" ng-repeat="element in selectedElement.stages track by $index">
+              <div class="col-xs-1">
+                <i class="fa fa-circle" style="color: {{getElementChartColor($index)}}" />
+              </div>
+              <div class="col-xs-5">
+                {{element.status | titleize}}
+              </div>
+              <div class="col-xs-4 text-right">
+                {{ element.date_range[0] + ' : ' + element.date_range[1] }}
+              </div>
+              <div class="col-xs-2 text-right">
+                <i>{{ element.duration + 'd' }}</i>
+              </div>
+            </div>
+          </div>
+
+        </div>
+
+        <div ng-hide="selectedElement" class="no-element">
+          Select one or several employee(s) or category(ies) to display the corresponding summary.
+        </div>
+
+      </div>
+
+    </div>
+
+    <div ng-show="(isDataFound==false)" common-data-not-found widget-engine="widget.category" />
```

Comment:
Please update this to `<div ng-show="(isDataFound==false)" common-data-not-found widget-engine="widget.category" width="widget.width"/>`, we have added a new feature recently that needs the widgets width passed into the `common-data-not-found` directive.